Maestro/subscriptions.json

```diff
@@ -1028,9 +1028,7 @@
     // Automate opening PRs to merge aspnet release/5.0-previewN branches back to master.
     {
       "triggerPaths": [
-        "https://github.com/dotnet/aspnetcore/blob/release/5.0/**/*",
         "https://github.com/dotnet/aspnetcore-tooling/blob/release/5.0/**/*",
-        "https://github.com/dotnet/efcore/blob/release/5.0/**/*",
         "https://github.com/dotnet/extensions/blob/release/5.0/**/*"
       ],
```

Comment:
Don't we need the `release/X.Y//**/*` workaround for the odd syntax in these files❔ Suspect a few of the newer `**` rules are not working correctly.

/cc @dagood